bigtable-dataflow-import/src/test/java/com/google/cloud/bigtable/dataflowimport/ImportIntegrationTest.java

```diff
@@ -0,0 +1,265 @@
+public class ImportIntegrationTest {
+
+  // Location of test data hosted on Google Cloud Storage, for on-cloud dataflow tests.
+  private static final String CLOUD_TEST_DATA_FOLDER = "cloud.test.data.folder";
+  private static final String HBASE_094_SUBFOLDER = "hbase094";
+  private static final String HBASE_CURRENT_SUBFOLDER = "post-hbase094";
+
+  // Column family name used in all test bigtables.
+  private static final byte[] CF = Bytes.toBytes("column_family");
+
+  // TestPipeline configurations.
+  private static final String RUN_INTEGRATION_TEST_ON_SERVICE = "runIntegrationTestOnService";
+  private static final String TEST_PIPELINE_OPTIONS = "dataflowOptions";
+
+  private HBaseImportOptions commonOptions;
+  private String cloudTestDataFolder;
+  private String svcAccountEmail;
+
+  @Rule
+  public TemporaryFolder tmpFolder = new TemporaryFolder();
+
+  @Before
+  public void setup() {
+    commonOptions = PipelineOptionsFactory.as(HBaseImportOptions.class);
+    String projectId =
+        checkNotNull(System.getProperty(BigtableOptionsFactory.PROJECT_ID_KEY), "Project Id");
+    commonOptions.setBigtableProjectId(projectId);
+    commonOptions.setBigtableClusterId(
+        checkNotNull(System.getProperty(BigtableOptionsFactory.CLUSTER_KEY), "Cluster Id"));
+    commonOptions.setBigtableZoneId(
+        checkNotNull(System.getProperty(BigtableOptionsFactory.ZONE_KEY), "Zone Id"));
+    commonOptions.setStagingLocation(
+        checkNotNull(System.getProperty("google.dataflow.stagingLocation"), "Staging Location"));
+
+    cloudTestDataFolder =
+        checkNotNull(System.getProperty(CLOUD_TEST_DATA_FOLDER), CLOUD_TEST_DATA_FOLDER);
+    if (!cloudTestDataFolder.endsWith(File.separator)) {
+      cloudTestDataFolder = cloudTestDataFolder + File.separator;
+    }
+  }
+
+  private HBaseImportOptions createImportOptions(
+      String importFilePath, String tableName, boolean isHBase094Format) {
+    HBaseImportOptions importOptions = commonOptions.as(HBaseImportOptions.class);
+    importOptions.setFilePattern(importFilePath);
+    importOptions.setBigtableTableId(tableName);
+    importOptions.setHBase094DataFormat(isHBase094Format);
+    if (importFilePath.startsWith("gs://")) {
+      importOptions.setRunner(BlockingDataflowPipelineRunner.class);
+    } else {
+      importOptions.setRunner(DirectPipelineRunner.class);
+    }
+    return importOptions;
+  }
+
+  @Test
+  public void testImportSequenceFile_onLocalFs() throws Exception {
+    String tableId = "test_" + UUID.randomUUID().toString();
+    File sequenceFile =tmpFolder.newFile(tableId);
+    Set<? extends Cell> expected = createTestData();
+    SequenceFileIoUtils.createFileWithData(sequenceFile, expected);
+
+    HBaseImportOptions options = createImportOptions(
+        sequenceFile.getPath(), tableId, false /** isHBase094Format **/);
+    BigtableTableUtils tableAdmin = BigtableTableUtils.from(options);
+    tableAdmin.createTable(Bytes.toString(CF));
```

Comment:
I'll restructure the class so that there is only one BigtableConnection and one Admin instance.

Just using one table for all tests requires rows purging before each test, which in hbase requires a scan and a one Mutation per row. Needs a bit more code and is not more efficient.